spark/src/main/scala/org/apache/spark/sql/delta/PreprocessTimeTravel.scala

```diff
@@ -43,15 +43,6 @@ case class PreprocessTimeTravel(sparkSession: SparkSession) extends Rule[Logical
   override def conf: SQLConf = sparkSession.sessionState.conf
 
   override def apply(plan: LogicalPlan): LogicalPlan = plan.resolveOperators {
-    case _ @ RestoreTableStatement(tt @ TimeTravel(ur @ UnresolvedRelation(_, _, _), _, _, _)) =>
-      val sourceRelation = resolveTimeTravelTable(sparkSession, ur)
-      return RestoreTableStatement(
-        TimeTravel(
-          sourceRelation,
-          tt.timestamp,
-          tt.version,
-          tt.creationSource))
-
     case ct @ CloneTableStatement(
```

Comment:
We can move the resolution of CloneTableStatement to ResolveDeltaIdentifier as well.